v2/cmd/wails/internal/commands/dev/dev.go

```diff
@@ -321,12 +331,21 @@ func loadAndMergeProjectConfig(cwd string, flags *devFlags) (*project.Project, e
 		shouldSaveConfig = true
 	}
 
-	if flags.devServerURL == defaultDevServerURL && projectConfig.DevServerURL != defaultDevServerURL && projectConfig.DevServerURL != "" {
-		flags.devServerURL = projectConfig.DevServerURL
+	if flags.devServer == "" && projectConfig.DevServer != "" {
+		flags.devServer = projectConfig.DevServer
+	}
+
+	if flags.devServer != projectConfig.DevServer {
+		projectConfig.DevServer = flags.devServer
+		shouldSaveConfig = true
```

Comment:
At some point, we should probably use a `-save` flag for the CLI. I own this pattern 😂